OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  KSConfig.swift
+//  KumulosSDK
+//
+//  Created by Andy on 05/10/2017.
+//  Copyright © 2017 Kumulos. All rights reserved.
+//
+
+import Foundation
+
+public struct OptimoveConfig {
+    let tenantInfo: OptimoveTenantInfo?
+    let optimobileConfig: OptimobileConfig?
+
+    internal func isOptimoveConfigured() -> Bool {
+        return tenantInfo != nil
+    }
+
+    internal func isOptimobileConfigured() -> Bool {
+        return optimobileConfig != nil
+    }
+}
+
+@objc public class OptimoveTenantInfo: NSObject {
+
+    @objc public var tenantToken: String
+    @objc public var configName: String
+
+    @objc public init(tenantToken: String, configName: String) {
+        self.tenantToken = tenantToken
+        self.configName = configName
+    }
+}
+
+public struct OptimobileConfig {
```

Comment:
For Objective-C interop we'll need to refactor these as NSObject classes. There's a separate task for the same.